blog/_src/posts/2017-11-16-monotonicity-types-towards-a-type-system-for-eventual-consistency.md

```diff
@@ -1,6 +1,6 @@
     Title: Monotonicity Types: Towards A Type System for Eventual Consistency
     Date: 2017-10-22T11:59:06
-    Tags: types, monotonicity, CRDTs, eventual consistency, by Kevin Clancy
+    Tags: types, monotonicity, CRDTs, eventual consistency, by Kevin Clancy and Heather Miller
```

Comment:
suggestion: change to `by Kevin Clancy, by Heather Miller`

(then you have 2 tags that can point to posts by 2 authors, instead of the 1 tag that points to co-authored posts)